settings/settings.go

```diff
@@ -227,3 +276,312 @@ func objectValidator(bound int, args ...string) bool {
+		if err != nil {
+			return errors.Wrap(err, "Writing variables to file unsuccessful")
+		}
+
+		defer f.Close()
+
+		for k, v := range m {
+			fmt.Println("Exporting Variable: " + k)
+			_, err := f.WriteString("\nAIRFLOW_VAR_" + strings.ToUpper(k) + "=" + v)
+			if err != nil {
+				fmt.Println(err)
+				fmt.Printf("error adding variable %s to file\n", k)
+			}
+		}
+		fmt.Println("Aiflow variables successfully export to the file " + envFile + "\n")
+		rmCmd := "rm tmp.var"
+		_ = execAirflowCommand(id, rmCmd)
+		return nil
+	}
+	return errors.New("variable export unsuccessful")
+}
+
+func EnvExportConnections(id, envFile string, logs bool) error {
+	// Airflow command to export connections to env uris
+	airflowCommand := "airflow connections export tmp.connections --file-format env"
+	out := execAirflowCommand(id, airflowCommand)
+	if logs {
+		fmt.Println("Env Export Connections logs:\n" + out)
+	}
+
+	if strings.Contains(out, "successfully") {
+		// get connections from file craeted by airflow command
+		fileCmd := "cat tmp.connections"
+		out = execAirflowCommand(id, fileCmd)
+
+		vars := strings.Split(out, "\n")
+		// add connections to the env file
+		f, err := os.OpenFile(envFile, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0o644) //nolint:gomnd
+		if err != nil {
+			return errors.Wrap(err, "Writing connections to file unsuccessful")
+		}
+
+		defer f.Close()
+
+		for i := range vars {
+			varSplit := strings.SplitN(vars[i], "=", 2)
+			if len(varSplit) > 1 {
+				fmt.Println("Exporting Connection: " + varSplit[0])
+				_, err := f.WriteString("\nAIRFLOW_CONN_" + strings.ToUpper(varSplit[0]) + "=" + varSplit[1])
+				if err != nil {
+					fmt.Println(err)
+					fmt.Printf("error adding connection %s to file\n", varSplit[0])
+				}
+			}
+		}
+		fmt.Println("Aiflow connections successfully export to the file " + envFile + "\n")
+		rmCmd := "rm tmp.connection"
+		_ = execAirflowCommand(id, rmCmd)
+		return nil
+	}
+	return errors.New("connection export unsuccessful")
+}
+
+func Export(id, settingsFile string, version uint64, connections, variables, pools, logs bool) error {
+	if id == "" {
+		return errNoID
+	}
+	// init settings file
+	err := InitSettings(settingsFile)
+	if err != nil {
+		return err
+	}
+	var parseErr bool
+	// export Airflow Objects
+	if version >= AirflowVersionTwo {
+		if pools {
+			err = ExportPools(id, logs)
+			if err != nil {
+				fmt.Println(err)
+				parseErr = true
+			}
+		}
+		if variables {
+			err = ExportVariables(id, logs)
+			if err != nil {
+				fmt.Println(err)
+				parseErr = true
+			}
+		}
+		if connections {
+			err := ExportConnections(id, logs)
+			if err != nil {
+				fmt.Println(err)
+				parseErr = true
+			}
+		}
+		if parseErr {
+			return errors.New("there was an error during export")
+		}
+		return nil
+	}
+
+	return errors.New("Command must be used with Airflow 2.X")
+}
+
+func ExportConnections(id string, logs bool) error {
```

Comment:
might be that I am missing something, but can I feel most of the logic of this method and `EnvExportConnections` are quite similar, can't we move that to common method let's say `GetConnections` and then this & `EnvExportConnections` task would be to just output the connections in desired format